app/controllers/neighborhoods_controller.rb

```diff
@@ -21,6 +22,22 @@ def create
     end
   end
 
+  def edit
+    @neighborhood = Neighborhood.find(params[:id])
+  end
+
+  def update
+    @neighborhood = Neighborhood.find(params[:id])
+    if @neighborhood.update_attributes(neighborhood_params) &&
+      current_user.id == @neighborhood.user_id
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.